app/jobs/data_warehouse/daily_sensitive_column_job.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module DataWarehouse
+  class DailySensitiveColumnJob < BaseJob
+    REPORT_NAME = 'daily-sensitive-column-job'
+
+    def perform(timestamp)
+      data = fetch_columns
+      upload_to_s3(data, timestamp)
+    end
+
+    def fetch_columns
+      tables = ActiveRecord::Base.connection.tables - %w[schema_migrations ar_internal_metadata]
+
+      sensitive_hash = []
+      insensitive_hash = []
+
+      tables.each do |table|
+        true_sensitives, false_sensitives = ActiveRecord::Base.connection.columns(table).
+          reject { |col| col.name == 'id' }.partition do |column|
+          column.comment&.match?(/sensitive=true/i)
+        end
```

Comment:
indentation suggestion so the `end` lines up with the block it's for
```suggestion
          reject { |col| col.name == 'id' }.
          partition do |column|
            column.comment&.match?(/sensitive=true/i)
          end
```

Reply:
This is fixed with the new commit